src/client/replication_ddl_client.cpp

```diff
@@ -1477,17 +1396,22 @@ void replication_ddl_client::end_meta_request(const rpc_response_task_ptr &callb
 dsn::error_code replication_ddl_client::get_app_envs(const std::string &app_name,
                                                      std::map<std::string, std::string> &envs)
 {
+    // Just match the table with the provided name exactly since we want to get the envs from
+    // a specific table.
     std::vector<::dsn::app_info> apps;
-    auto r = list_apps(dsn::app_status::AS_AVAILABLE, apps);
-    if (r != dsn::ERR_OK) {
-        return r;
+    const auto &result = list_apps(
```

Comment:
There are many code snippet like this, we can wrap it by a macro in next patch to simplify the code.
```
    auto a = xxx;
    if (!a) {
        return a.code();
    }
```

Reply:
OK, define `RETURN_EC_NOT_OK` to wrap it.